cmd/installer/pkg/install/manifest.go

```diff
@@ -550,6 +546,10 @@ func (t *Target) Partition(pt *gpt.GPT, pos int, bd *blockdevice.BlockDevice) (e
 		return err
 	}
 
+	if err = pt.Write(); err != nil {
```

Comment:
why are we writing partition table here?

Reply:
I think I had some idea about it, but I no longer remember :shrug: Removed